source/_integrations/squeezebox.markdown

```diff
@@ -82,6 +87,10 @@ Volume step:
  description: Amount to adjust the volume when turning volume up or down.  
 {% endconfiguration_basic %}
 
+## Known limitations
+
+The LMS API, which is used by this integration, does not currently provide the ability to override or control fade-in & crossfade settings. This means that if you have enabled **Play or Resume fade-in duration** within the player's audio settings, this fade-in will be applied to any announcement played.  This could potentially lead to the start of an announcement being missed as it fades in.  You should, therefore, consider a short **Play or Resume fade-in duration** or preferably disabling this feature if you make use of announcements.
+
 ## Removing the integration
```

Comment:
I would move the removal section to the end to the page, as it marks the end of the integration's life-cycle.